ivy/functional/frontends/torch/linalg.py

```diff
@@ -64,6 +64,15 @@ def matrix_power(input, n, *, out=None):
     return ivy.matrix_power(input, n, out=out)
 
 
+@to_ivy_arrays_and_back 
+def vector_norm(
+    input, ord=2, dim=None, keep_dim=False, *, dtype=None, out=None
+):
+    return ivy.vector_norm(
+        input, ord=ord, dim=dim, keep_dim=keep_dim, dtype=dtype, out=out
```

Comment:
`ivy.vector_norm` does not take a `dim` parameter, you will need to replicate the behavior of this parameter either by adding its functionality to ivy.vector_norm, or in your own frontend function. Either one would be sufficient for passing this PR! Thanks :)